src/hdf5/tropics_2ioda.py

```diff
@@ -335,9 +344,15 @@ def get_WMO_satellite_ID(attrs_shortname):
         WMO_sat_ID = TROPICS06_WMO_sat_ID
     elif 'TROPICS07' in attrs_shortname:
         WMO_sat_ID = TROPICS07_WMO_sat_ID
+    elif 'TMS_S3_BRTTL1B' in attrs_shortname:
+        WMO_sat_ID = TOMORROWIO_S3_sat_ID
+    elif 'TMS_S1_BRTTL1B' in attrs_shortname:
+        WMO_sat_ID = TOMORROWIO_S1_sat_ID
+    elif 'TMS' in attrs_shortname and 'BRTTL1B' in attrs_shortname:
+        WMO_sat_ID = TOMORROWIO_GENERIC_sat_ID
```

Comment:
prefer to use this, does tomorrow have a single or multiple WMO satellite IDs ... or any at all yet?